endpoints/orders.yaml

```diff
@@ -290,229 +290,71 @@ paths:
                 schema:
                   $ref: 'schemas/commonSchemas.yaml#/components/schemas/Error'
           
-  /v1/orders/subscribe:
+  /v1/orders/subscriptions:
       post:
-          summary: Subscribe to order events
-          description: Subscribe to order events
-          operationId: ordersSubscribe
+          summary: Create a new subscription
+          description: Create a new subscription
+          operationId: ordersSubscription
           requestBody:
-            description: ordersSubscribe
+            description: Create a new subscription
             required: true
             content:
               application/json:
                 schema:
-                  oneOf:
-                    - $ref: 'schemas/ordersSchemas.yaml#/components/schemas/ordersSubscribeSGS'
-                    - $ref: 'schemas/ordersSchemas.yaml#/components/schemas/ordersSubscribeSGSCounterParty'
+                  $ref: 'schemas/commonSchemas.yaml#/components/schemas/subscriptionRequest'
           responses:
             '202':
```

Comment:
same here I guess, `200`, not `202`

Reply:
as per JC's description yaml should be 201.